data-products/_docs/product-governance.md

```diff
@@ -92,9 +92,9 @@ It is important to users of the data that they know where it has come from. This
 - `data-platform.justice.gov.uk/organisational-unit` defines at a high level which part of the organisation the data has come from. For example "HMCTS", "HMPPS", "LAA", "OPG", "HQ"
 - `data-platform.justice.gov.uk/source` - the detailed source of the data. Can be a system such as "NOMIS" or "XHIBIT" or the URL of a statistical publication
 - \- **OR** \-
-- `data-platform.justice.gov.uk/parent-product` - if your data product is based on another data product, please specify its name here. (The "name" as it appears in the list of data platform product folders, not the user-friendly title of the product).
+- `data-platform.justice.gov.uk/parent-products` - if your data product is based on another data product(s), please specify their name here. (The "name" as it appears in the list of data platform product folders, not the user-friendly title of the product).
```

Comment:
How do you envisage multiple products? I'd suggest allowing either a single string (one product name) or an array of product names
`data-platform.justice.gov.uk/parent-products: "parent_product"`
OR
`data-platform.justice.gov.uk/parent-products: ["parent_product", "other_parent_product"]`

"specify their name" => "specify their name(s)"

Reply:
The latter, with an array of names

Reply:
Might be worth making that clear in the doc?